src/current/v24.3/sso-sql.md

```diff
@@ -12,7 +12,7 @@ Cluster single sign-on (SSO) enables users to access the SQL interface of a Cock
 {{ site.data.products.advanced }} clusters can provision their users with JWTs via the DB Console. This allows users to authenticate to a cluster by signing in to their IdP (for example, Okta or Google) with a link embedded in the DB Console. This flow provisions a JWT that a user can copy out of the DB Console UI and use in a SQL connection string to authenticate to the cluster.
 
 {{site.data.alerts.callout_info}}
-Cluster single sign-on for the DB Console is supported on {{ site.data.products.core }}, {{ site.data.products.enterprise }} and {{ site.data.products.advanced }} clusters. {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }} clusters do not support cluster single sign-on. However, both CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }} clusters can use [Cluster Single Sign-on (SSO) using `ccloud` and the CockroachDB Cloud Console]({% link cockroachcloud/cloud-sso-sql.md %}).
+Cluster single sign-on for the DB Console is supported on {{ site.data.products.core }} {{ site.data.products.enterprise }} and {{ site.data.products.advanced }} clusters. {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }} clusters do not support cluster single sign-on. However, both CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }} clusters can use [Cluster Single Sign-on (SSO) using `ccloud` and the CockroachDB Cloud Console]({% link cockroachcloud/cloud-sso-sql.md %}).
```

Comment:
This displays as: `Cluster single sign-on for the DB Console is supported on Self-Hosted Enterprise and Advanced clusters.`. However, the language is ambiguous because it could read as "self-hosted Enterprise and self-hosted Advanced clusters". It seems simplest to remove the `core` variable here so that it says `Cluster single sign-on for the DB Console is supported on Enterprise and Advanced clusters.` (especially since the `core` variable is currently misleading and should prob be renamed; I'll raise that to the team). 

Alternatively, we have it say `Cluster single sign-on for the DB Console is supported on self-hosted and Advanced clusters.` where `self-hosted` (lowercase) is displayed by a new product variable that means both Enterprise and Enterprise Free. To me, this is preferable but perhaps it's out of scope for this PR, which is about removing Core. I'll raise this separately also.

Links to the appropriate license/product pages would also help clarify this section, in case people need a refresher on what all these product names mean. Not sure what the best links would be?

```suggestion
Cluster single sign-on for the DB Console is supported on CockroachDB {{ site.data.products.enterprise }} and {{ site.data.products.advanced }} clusters. CockroachDB {{ site.data.products.standard }} and {{ site.data.products.basic }} clusters do not support cluster single sign-on. However, both CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }} clusters can use [Cluster Single Sign-on (SSO) using `ccloud` and the CockroachDB {{ site.data.products.cloud }} Console]({% link cockroachcloud/cloud-sso-sql.md %}).
```

Reply:
Your suggested change appears to have been ~resolved by @mdlinville's changes in https://github.com/cockroachdb/docs/pull/19003 

It now reads 

> Cluster single sign-on for the DB Console is supported on {{ site.data.products.enterprise }} and {{ site.data.products.advanced }} clusters. {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }} clusters do not support cluster single sign-on and do not have access to the DB Console. However, both CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }} clusters can use [Cluster Single Sign-on (SSO) to authenticate to the `ccloud` command-line interface and to the CockroachDB Cloud Console]({% link cockroachcloud/cloud-sso-sql.md %})

I will rebase on `main` to pick this up